doc/manual/meson.build

```
@@ -0,0 +1,75 @@
+if get_option('doc')
+  ignore_headers = [
+    'private.h',
+    'pkcs11.h',
+    'conf.h',
+    'debug.h',
+    'dict.h',
+    'filter.h',
+    'log.h',
+    'mock.h',
+    'modules.h',
+    'pkcs11.h',
+    'pkcs11i.h',
+    'pkcs11x.h',
+    'private.h',
+    'client.h',
+    'proxy.h',
+    'rpc.h',
+    'rpc-message.h',
+    'util.h',
+    'virtual.h',
+    'virtual-fixed.h',
+    'array.h',
+    'compat.h'
+  ]
+
+  content_files = [
+    'p11-kit-config.xml', 'p11-kit-sharing.xml',
+    'p11-kit-devel.xml',
+    'p11-kit-proxy.xml',
+    'p11-kit-trust.xml',
+    'p11-kit-remoting.xml',
+    'p11-kit.xml',
+    'pkcs11.conf.xml',
+    'trust.xml',
+    'annotation-glossary.xml'
+  ]
+
+  version_conf = configuration_data()
+  version_conf.set('VERSION', meson.project_version())
+  configure_file(
+    input : 'version.xml.in',
+    output : 'version.xml',
+    configuration : version_conf)
+
+  userdir_conf = configuration_data()
+  userdir_conf.set('p11_user_config', p11_user_config)
+  configure_file(
+    input : 'userdir.xml.in',
+    output : 'userdir.xml',
+    configuration : userdir_conf)
+
+  sysdir_conf = configuration_data()
+  sysdir_conf.set('p11_system_config', p11_system_config)
+  configure_file(
+    input : 'sysdir.xml.in',
+    output : 'sysdir.xml',
+    configuration : sysdir_conf)
+
+  gnome.gtkdoc('p11-kit',
+               main_xml : 'p11-kit-docs.xml',
+               namespace : 'p11_kit',
+               src_dir : 'p11-kit',
+               dependencies : libffi_deps + libdl_deps,
+               scan_args : [
+                 '--ignore-headers=' + ' '.join(ignore_headers),
+                 '--rebuild-types',
+                 '--deprecated-guards="P11_KIT_DISABLE_DEPRECATED"'
+               ],
+               scanobjs_args : [
```

There is no need to use an array if only one parameter is used.

```suggestion
               scanobjs_args : '--version',
```